### DISCUSSION TITLE
Hafizilib'e Hoş Geldin! 👋

### DISCUSSION THREAD

**Discussion by ugurkilci** [ugurkilci/hafizilib#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HACKERA316** [comment]
Hocam öncelikle bu proje yaptığınız için teşekkürler fakat dosya kullanımı fazla iyi değil php,css,js (vb) bunları ayrı bir klasörde kullanmak daha iyi olur ve kullanıcı acısından göze daha iyi gelir

**Reply by ugurkilci** [reply]
bunu düşüneceğim
